feedback/templates/public/create-survey.html

```diff
@@ -51,14 +66,14 @@
 				<div class='row twelve columns'>
 					
 					<div class='two columns'>.</div>
-					<div class='eight columns'>
+					<div class='eight columns' id='answerType'>
```

Comment:
Probably. For now let's leave it as-is -- I had a reason for using ID, though I can't currently remember what it was!!